INSTALL.txt

```diff
@@ -138,18 +160,18 @@ together toolchains for that option. The toolchains are available at
 https://bitbucket.org/carlkl/mingw-w64-for-python/downloads. The site.cfg
 should be configured like so:
```

Comment:
This is a .txt file, not reST. So double-colon doesn't do anything. Current style in the file is also single colon.